app/helpers/settings_helper.rb

```diff
@@ -152,4 +132,37 @@ def wrap_field_outer(options, &block)
       block.call
     end
   end
+
+  def build_settings_matrix_head(settings, options = {})
+    content_tag(:thead) do
+      content_tag(:tr, class: 'form--matrix-header-row') do
+        content_tag(:th, I18n.t(options[:label_choices] || :label_choices),
+                    class: 'form--matrix-header-cell') +
+          settings.map do |setting|
+            content_tag(:th, class: 'form--matrix-header-cell') do
+              hidden_field_tag("settings[#{setting}][]", '') +
+                I18n.t("setting_#{setting}")
+            end
+          end.join.html_safe
+      end
+    end
+  end
+
+  def build_settings_matrix_body(settings, choices)
```

Comment:
Assignment Branch Condition size for build_settings_matrix_body is too high. [15.33/15]